yeelight/main.py

```diff
@@ -59,15 +61,25 @@ def _command(f, *args, **kw):
     if result:
         return result[0]
 
+def get_ip_address(ifname):
```

Comment:
Can you add a quick docstring here to explain what this does, for posterity?

Reply:
Well, that was a shameless Stackoverflow copy-paste, it's way over my head 😅https://stackoverflow.com/questions/24196932/how-can-i-get-the-ip-address-of-eth0-in-python

Reply:
Haha, story of my life.